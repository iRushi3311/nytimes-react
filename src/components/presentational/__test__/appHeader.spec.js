import React from 'react';
import renderer from 'react-test-renderer';
import AppHeader from '../app-header.js';

describe('>>>C O M P O N E N T S --- snapshot test for AppHeader', () => {

    let componentTree;

    beforeEach(() => {
        componentTree = renderer.create(<AppHeader />).toJSON();
    });

    it('+++ AppHeader match with snapshot ', () => {
        expect(componentTree).toMatchSnapshot();
    });

});