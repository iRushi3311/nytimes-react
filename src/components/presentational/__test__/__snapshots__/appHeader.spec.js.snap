// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>>C O M P O N E N T S --- snapshot test for AppHeader +++ AppHeader match with snapshot  1`] = `
<div
  className="sc-htpNat bRqiJX"
>
  <div
    className="sc-bxivhb iQkVpv"
  >
     
    THE TIMES
     
  </div>
  <div
    className="pull-right sc-bdVaJa bEfZTs"
  >
    <form
      onSubmit={[Function]}
    >
      <div
        className="sc-bwzfXH eGbUHl form-group"
      >
        <span
          className="input-group"
        >
          <span
            className="override-border input-group-addon"
          >
            <span
              className="glyphicon glyphicon-search"
            />
          </span>
          <input
            className="override-border form-control"
            id={undefined}
            onChange={[Function]}
            placeholder="Search"
            type="text"
          />
        </span>
      </div>
    </form>
  </div>
</div>
`;
